app/uk/gov/hmrc/customs/emailfrontend/model/VerifiedEmailRequest.scala

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.emailfrontend.model
+
+import java.time.Clock
+import java.util.UUID
+
+import org.joda.time.{DateTime, DateTimeZone}
+import org.joda.time.format.ISODateTimeFormat
+import play.api.libs.json.{JsError, JsResult, JsString, JsSuccess, JsValue, Json, Reads, Writes}
+
```

Comment:
can we split the models into different files specific to model, to reduce number of lines in one file

Reply:
Also, might worth adding tests for generic/common code like `dateTimeWritesIsoUtc`, `dateTimeReadsIso`, `MDGDateFormat`, and others